libr/util/lib.c

```diff
@@ -306,6 +306,11 @@ R_API int r_lib_open_ptr (RLib *lib, const char *file, void *handler, RLibStruct
 
 R_API int r_lib_opendir(RLib *lib, const char *path) {
 	char file[1024];
+#if __WINDOWS__
+	WIN32_FIND_DATA dir;
+	HANDLE fh;
+	char directory[256];
```

Comment:
Windows uses `wchar_t` in their API to support unicode and the (fake) `MAX_PATH` is 260 characters not 256...